geocodingsearchv7/client.go

```diff
@@ -24,6 +25,9 @@ type GeocodingService service
 // ReverseGeocodingService handles communication with reverse geocoding-related methods of the v7 HERE API.
 type ReverseGeocodingService service
 
+// BatchGeocoderService handles communication with batch geocoder-related methods of the v7 HERE API.
+type BatchGeocoderService service
```

Comment:
Some inconsistency in naming here
```suggestion
// BatchGeocodingService handles communication with batch geocoding-related methods of the v7 HERE API.
type BatchGeocodingService service
```